bottlecap/src/logs/agent.rs

```diff
@@ -4,12 +4,12 @@ use std::thread;
 
 use tracing::{debug, error};
 
-use crate::config;
 use crate::logs::aggregator::Aggregator;
 use crate::logs::datadog;
-use crate::logs::lambda::processor::LambdaProcessor;
-use crate::tags::provider;
+use crate::logs::processor;
```

Comment:
Can we consolidate logs imports?

This could be:
```suggestion
use crate::logs::{datadog, aggreagtor::Aggregator, processor};
```